src/orbit-provider.ts

```diff
@@ -0,0 +1,297 @@
+  const seneca: any = this
+
+  const entityBuilder = this.export('provider/entityBuilder')
+
+  seneca
+    .message('sys:provider,provider:orbit,get:info', get_info)
+
+
+
+  const makeUrl = (suffix: string, q: any) => {
+    let url = options.url + suffix
+    if (q) {
+      if ('string' === typeof q) {
+        url += '/' + encodeURIComponent(q)
+      }
+      else if ('object' === typeof q && 0 < Object.keys(q).length) {
+        url += '?' + Object
+          .entries(q)
+          .reduce(((u: any, kv: any) =>
+            (u.append(kv[0], kv[1]), u)), new URLSearchParams())
+          .toString()
+
+      }
+    }
+    return url
+  }
+
+  const makeConfig = (config?: any) => seneca.util.deep({
+    headers: {
+      ...seneca.shared.headers
+    }
+  }, config)
+
+
+  const setWorkspace = (newWorkspace: string) => {
+    workspace = newWorkspace;
+  }
+
+  const getWorkspace = () => {
+    return workspace;
+  }
+
+  const getJSON = async (url: string, config?: any) => {
+    let res = await options.fetch(url, config)
+    if (200 == res.status) {
+      let json: any = await res.json()
+      return json
+    }
+    else {
+      let err: any = new Error('OrbitProvider ' + res.status)
+      err.orbitResponse = res
+      throw err
+    }
+  }
+
+
+  const postJSON = async (url: string, config: any) => {
+    config.body = 'string' === typeof config.body ? config.body : JSON.stringify(config.body)
+
+    config.headers['Content-Type'] = config.headers['Content-Type'] ||
+      'application/json'
+
+    config.method = config.method || 'POST'
+
+    let res = await options.fetch(url, config)
+
+    if (201 === res.status || 200 === res.status ) {
+      let json: any = await res.json()
+      return json
+    }
+    else {
+      let err: any = new Error('OrbitProvider ' + res.status)
+      try {
+        err.body = await res.json()
+      }
+      catch (e: any) {
+        err.body = await res.text()
+      }
+      err.status = res.status
+      throw err
+    }
+  }
+
+  const putJSON = async (url: string, config: any) => {
+    config.body = 'string' === typeof config.body ? config.body :
+    JSON.stringify(config.body)
+   
+
+    config.headers['Content-Type'] = config.headers['Content-Type'] ||
+      'application/json'
+
+    config.method = config.method || 'PUT'
+    let res = await options.fetch(url, config)
+
+    if (204 <= res.status) {
+      return {"Success": true}
+    }
+    else {
+      let err: any = new Error('OrbitProvider ' + res.status)
+      try {
+        err.body = await res.json()
+      }
+      catch (e: any) {
+        err.body = await res.text()
+      }
+      err.status = res.status
+      throw err
+    }
+  }
+
+
+  async function get_info(this: any, _msg: any) {
+    return {
+      ok: true,
+      name: 'orbit',
+      version: Pkg.version,
+    }
+  }
+
+
+  entityBuilder(this, {
+    provider: {
+      name: 'orbit'
+    },
+    entity: {
+      list_member: {
```

Comment:
rename this to `member` and move the operations into the `list`,`save`,`load` operations - we need to follow the structure of REST operations

Reply:
i believe now this correct